src/bun.js/net.exports.js

```diff
@@ -190,6 +191,17 @@ export const Socket = (function (InternalSocket) {
       return this.writableLength;
     }
 
+    _attach(port, socket) {
```

Comment:
Is this a public function Node.js exposes? If it's not a public function, it should be a #private function

Reply:
A symbol should be the right call here like I'm static _Handlers